Sources/Signal.swift

```diff
@@ -1677,9 +1677,13 @@ extension SignalProtocol {
 			return disposable
 		}
 	}
-
-	/// Throttle values sent by the receiver, so that at least `interval`
-	/// seconds pass between each, then forwards them on the given scheduler.
+	
+	/// Don't send a value until at least `interval` seconds have passed since
+	/// *the returned signal* last sent a value, then forward the latest value
+	/// on `scheduler`.
+	///
+	/// If the receiver sent values constantly, then the returned signal would
```

Comment:
This is a tricky thing in general, but "the receiver" is difficult to use as a noun in concert with "sending and receiving" values.

"The receiver" is Objective-C terminology where an object "receives" a message, but now we're "calling functions on" objects. Had Swift instead used `mesg` rather than `func` then I'd be more OK with this old terminology. But I digress!

You might be able to safely use this wording instead:

```
/// Returns a signal that sends values exactly, or less frequently than, `interval` seconds 
/// apart, forwarding values on the given `scheduler`.
///
/// That is, the returned signal does not send a value until at least `interval` seconds have
/// passed since it last sent a value.
```